state/spaces.go

```diff
@@ -191,7 +236,7 @@ func (s *Space) EnsureDead() (err error) {
 
 	ops := []txn.Op{{
 		C:      spacesC,
-		Id:     s.doc.Name,
```

Comment:
These are also places where "DocId" is actually better.
I'm guessing what is accidentally happening is that because txn.Op is referencing by an Id, it ends up overwriting _id to make the txn.Op (and expanding to <model-uuid:id> because of multiModelRunner). But it would be better to be explicit, IMO.
I personally don't like txnRunner overwriting the _id field, as it adds confusion between what the Insert: doc is and what actually gets put into the database.